### DISCUSSION TITLE
CldUploadWidget : Cropping but prevent from scaling

### DISCUSSION THREAD

**Discussion by richard-chang-coder** [cloudinary-community/next-cloudinary#518] [Q&A] [answered]
Hi, 

For the CldUploadWidget, is there a way from letting the user scaling up or down when you using the cropping? I would like to have croppingAspectRatio as 1 and prevent user to have the ability scale the box bigger or smaller. 

Thanks

**Comment by colbyfayock** [comment, marked as answer]
hey @richard-chang-coder it looks like you're able to configure a specific aspect ratio, you can check out the Upload Widget cropping parameters herE: https://cloudinary.com/documentation/upload_widget_reference#cropping_parameters

`croppingAspectRatio`

is that what you had in mind?

**Reply by richard-chang-coder** [reply]
Thank you , I think found the solution
